server/service/integration_enterprise_test.go

```diff
@@ -10722,28 +10724,22 @@ func (s *integrationEnterpriseTestSuite) TestListHostSoftware() {
 	getDeviceSw = getDeviceSoftwareResponse{}
 	err = json.NewDecoder(res.Body).Decode(&getDeviceSw)
 	require.NoError(t, err)
-	require.Len(t, getDeviceSw.Software, 3) // foo, bar and ruby
+	require.Len(t, getDeviceSw.Software, 2) // foo, bar and ruby
```

Comment:
Ruby no longer shows up because 
```
	*{
		ID: 4,
		Name: "ruby",
		Source: "deb_packages",
		Status: *"pending_install",
		InstalledVersions: []*github.com/fleetdm/fleet/v4/server/fleet.HostSoftwareInstalledVersion len: 0, cap: 0, nil,
		SoftwarePackage: *(*"github.com/fleetdm/fleet/v4/server/fleet.SoftwarePackageOrApp")(0x140000df3f0),
		AppStoreApp: *github.com/fleetdm/fleet/v4/server/fleet.SoftwarePackageOrApp nil,},

```

Reply:
```suggestion
	require.Len(t, getDeviceSw.Software, 2) // foo and bar; ruby doesn't show up as it is still pending
```